### PR TITLE
Bump the dependency on ouroboros-network and port recent changes

### PR DESCRIPTION
**Pull request by mrBliss**
As ouroboros-network bumps its dependencies on some packages as well as
GHC (8.6.4 -> 8.6.5), we have to follow suit.

### CODE REVIEW

**Comment by mrBliss** [comment]
Pushed (no changes) again to trigger CI.

**Comment by Jimbo4350** [comment]
We should probably check that the demo scripts are still working

File: scripts/submit-tx.sh [hunk 1]
```diff
@@ -5,5 +5,6 @@ now=`date "+%Y-%m-%d 00:00:00"`
 set -x
 cabal new-run cardano-node -- \
     --system-start "$now" --slot-duration 2 \
+    --log-config configuration/log-configuration.yaml \
```
Comment (mrBliss) [inline scripts/submit-tx.sh]: Actually, this should not really be needed for submitting a transaction.

**Pull request merged**